File: src/main/java/com/tngtech/jgiven/Icons.java
package com.tngtech.jgiven;

import com.intellij.openapi.util.IconLoader;

import javax.swing.*;

public class Icons {
    private static final String JGIVEN_PNG = "/icons/jgiven.png";

    public static final Icon JGIVEN = load(JGIVEN_PNG);

    private static Icon load(String path) {
        return IconLoader.getIcon(path, Icons.class);
    }
}

File: src/main/java/com/tngtech/jgiven/Icons.kt
package com.tngtech.jgiven

import com.intellij.openapi.util.IconLoader

object Icons {
    private const val JGIVEN_PNG = "/icons/jgiven.png"
    val JGIVEN = load(JGIVEN_PNG)

    private fun load(path: String) = IconLoader.getIcon(path, Icons::class.java)
}
